helm-chart/splunk-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,89 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "splunk-operator.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "splunk-operator.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/* 
+Create standard operator fullname for uniform installation across Helm and manifest
+*/}}
+{{- define "splunk-operator.operator.fullname" -}}
+{{- default "splunk-operator" .Values.splunkOperator.nameOverride }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use for splunk operator TODO: issue if create not selected
```

Comment:
Has this TODO been taken care of?

Reply:
Yes it has, thank you for catching that. I've cleaned up the `_helpers.tpl` file to reflect that.